services/venice-router/src/main/java/com/linkedin/venice/router/api/path/VeniceSingleGetPath.java

```diff
@@ -32,18 +35,17 @@ public class VeniceSingleGetPath extends VenicePath {
       Integer.toString(ReadAvroProtocolDefinition.SINGLE_GET_ROUTER_REQUEST_V1.getProtocolVersion());
 
   private final RouterKey routerKey;
-  private final String partition;
 
   public VeniceSingleGetPath(
-      String storeName,
-      int versionNumber,
-      String resourceName,
+      StoreVersionName storeVersionName,
```

Comment:
This is a very satisfying compaction of arguments.

storeName, versionNumber, resourceName -> storeVersionName

Reply:
One of those cases where doing the efficient thing is also cleaner (more brief, strongly-typed) 😁